src/docs/documentation.ts

```diff
@@ -131,4 +130,96 @@
  *         description: User not found
  *       500:
  *         description: Error occurred while deleting user
- */
\ No newline at end of file
+ *
+ */
+
+/**
+ * @openapi
+ * /api/v1/auth/forgot-password:
```

Comment:
This is good but untidy. Let's put every major feature(controller) with its endpoints in its own file. for example: `auth.docs.ts`, `user.docs.ts`, `product.docs.ts`